packages/api-prerendering-service-so-ddb/tsconfig.json

```diff
@@ -1,35 +1,29 @@
 {
-  "extends": "../../tsconfig",
+  "extends": "../../tsconfig.json",
```

Comment:
BTW what did cause the items to be alphabetically sorted? Some tool or you did this manually?

Reply:
We have a script that generates all tsconfigs. I simply run it.

Reply:
Cool.